.github/workflows/yaml-validation.yml

```diff
@@ -0,0 +1,41 @@
+name: YAML validation
+
+on:
+  push:
+    branches: [ devel ]
+  pull_request:
+    branches: [ master, devel ]
+
+jobs:
+  yaml-validation:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+
+    - name: YAML Lint
+      uses: ibiqlik/action-yamllint@v1.0.0
+      continue-on-error: true
+      with:
+        ## File(s) or Directory, separate by space if multiple files or folder are specified
+        file_or_dir: ModelFiles/
+        ## Custom configuration (as YAML source)
+        config_data: "{extends: default, rules: {line-length: disable}}"
+        ## Format for parsing output [parsable,standard,colored,auto]
+        # format: # optional, default is colored
+        ## Return non-zero exit code on warnings as well as errors
+        # strict: # optional, default is false
+
+    - name: Set up Python 3
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Import with cobrapy
+      continue-on-error: true
+      run: |
+        pip install cobra
```

Comment:
The idea behind using `pip-tools` was to avoid things like `pip install cobra` by having a frozen environment that is updated sporadically. This way we can separate build fails due to a model change, from build fails due to an updated version of cobra (which is a common problem I have faced with e.g. geckopy). We would then encounter the latter build fails in an isolated way whenever we re-compile dependencies.

> installation of `requirements/requirements.txt` fails because of a dependency on `pywin32`

`requirements/requirements.txt` is meant for users, therefore it also includes dependencies for Jupyter Notebook, that the CI does not require. How about having cobra in a `requirements/ci-requirements.in` file, and using `pip-compile` in a ubuntu env to create `requirements/ci-requirements.txt`?